### DISCUSSION TITLE
Welcome to RLE-wiki Discussions!

### DISCUSSION THREAD

**Discussion by HalfSweet** [project-trans/RLE-wiki#142] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ddzzkunye** [comment]
请问, 是否有金陵科技学院, 南京晓庄学院的学长能够提供信息的?
